app/assets/javascripts/controllers/ems_common/arbitration_profile_form_controller.js

```diff
@@ -79,9 +82,9 @@ ManageIQ.angular.app.controller('arbitrationProfileFormController', ['$scope', '
   var emsId = (/ems_cloud\/arbitration_profile_edit\/(\d+)/.exec($location.absUrl())[1]);
   var redirectUrl = '/ems_cloud/arbitration_profiles/' + emsId + '?db=ems_cloud';
 
-  var convertToString = function(id) {
+  var setRecordId = function(id) {
```

Comment:
But not sure why we actually need it, having `null` (or `undefined`) id if not given would make more sense than en empty string, right?